curriculum/challenges/english/16-the-odin-project/top-the-box-model/the-box-model-question-h.md

```diff
@@ -0,0 +1,36 @@
+---
+id: 64a674937a7d00f97013ed0f
+title: The Box Model Question H
+challengeType: 15
+dashedName: the-box-model-question-h
+--- 
+# --description--
+
+Because the box model concept is so incredibly fundamental, let’s dig a bit deeper with [this lesson from MDN](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/The_box_model#what_is_the_css_box_model). It covers the same material as the video above and will introduce you to inline boxes that we will explore in the next lesson. Pay close attention to the examples and take the time to experiment with their in-browser editor!
```

Comment:
It says, `It covers the same material as the video above`. There's no video above. Maybe we should make this a video challenge type?